content/updates/2017-08-30-release-notes-plus-join-september-workshop.md

```diff
@@ -0,0 +1,45 @@
++++
+date = "2017-08-30"
+title = "cloud.gov platform release notes, plus join our workshop on September 28"
++++
+Curious what’s new that you might find helpful as a cloud.gov user? Here are highlights from our platform updates over the past two weeks.
+
+### Announcements
+
+[**cloud.gov Workshop:** You’re invited to our free cloud.gov workshop on September 28th](https://www.digitalgov.gov/event/hands-on-workshop-with-cloud-gov/). If you’re evaluating cloud.gov or getting started with it, you will learn how to launch applications and see the power and advantages that come from using a Platform as a Service. Whether you’re in-person or online, bring your laptop and prepare to learn how cloud.gov can work for you.
```

Comment:
I'm going to nit-pick a bit:   Needs parallel structure or it reads like you'll "learn how to ... see the power and advantages"  Here's the wording I used in an email:

> Join us for this walk-through of how to release a cloud.gov web application, connect it to a database, and maintain the application’s security and compliance.  Participants will come away understanding how the cloud.gov Platform-as-a-Service model supports rapid development and continuous compliance better than other infrastructure approaches. 

Bring a laptop if you're online doesn't quite work. How about:

> Bring your laptop for the in-person experience, or boot up your workstation if you're remote, and prepare to learn how cloud.gov can work for you.